dysnix/op-node/values.yaml

```diff
@@ -0,0 +1,162 @@
+image:
+  repository: us-docker.pkg.dev/oplabs-tools-artifacts/images/op-node
+  tag: ""
+  pullPolicy: Always
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+updateStrategy:
+  type: RollingUpdate
+
+terminationGracePeriodSeconds: 300
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+  # runAsGroup: 1000
+
+## Override op-node command (can be templated)
+command: []
+
+## Extra op-node arguments (can be templated)
+extraArgs: []
+
+## Extra init containers, can be templated
+initContainers: []
+
+## Sidecar containers, can be templated
+sidecarContainers: []
+
+## Services config
+services:
+  p2p:
+    enabled: true
+    type: NodePort
+    loadBalancerIP: ""
+    port: 9222
+    # it's better to set nodePort equal to .Values.config.node.p2p.port when the svc type is "NodePort"
+    # nodePort: 9222
+    annotations: {}
+    publishNotReadyAddresses: true
+  p2pDiscovery:
+    enabled: true
+    type: NodePort
+    loadBalancerIP: ""
+    port: 9222
+    # it's better to set nodePort equal to .Values.config.node.p2p.port when the svc type is "NodePort"
+    # nodePort: 9222
+    annotations: {}
+    publishNotReadyAddresses: true
+  rpc:
+    enabled: true
+    type: ClusterIP
+    port: 9545
+    annotations: {}
+  metrics:
+    enabled: true
+    type: ClusterIP
+    port: 7300
+    annotations: {}
+    publishNotReadyAddresses: true
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+    # cert-manager.io/cluster-issuer: letsencrypt-prod
+  hosts: []
+    # - host: op-node.examplerpc.com
+    #   paths:
+    #     - path: /
+    #       pathType: ImplementationSpecific
+  tls: []
+    # - secretName: op-node-tls
+    #   hosts:
+    #     - op-node.examplerpc.com
+
+# Create Prometheus Operator serviceMonitor
+serviceMonitor:
+  enabled: false
+  # interval: 10s
+  # scrapeTimeout: 2s
+  # honorLabels: true
+  # relabelings: []
+  # metricRelabelings: []
+
+livenessProbe:
```

Comment:
Do we expect to enable liveness and readiness probes by default in upcoming iterations?

Reply:
Yes, will be in future releases. Though I can't exactly tell how the 'readinessProbe' will look like for op-node.
It is not intended to expose op-node's public RPC endpoint for now.

Reply:
For now enabled livenessProbe by default. readinessProbe to be implemented in next releases, if needed